src/alert/alert.ts

```diff
@@ -60,11 +60,16 @@ export class NgbAlert {
  * Alerts that can be dismissed without any additional code.
  */
 @Directive({selector: 'template[ngbAlert]'})
-export class NgbDismissibleAlert implements OnInit, OnDestroy {
+export class NgbSelfClosingAlert implements OnInit, OnDestroy {
   private _popupService: PopupService<NgbAlert>;
   private _timeout;
 
   /**
+   * A flag indicating if the alert can be dismissed (closed) by a user. If this flag is set, a close button (in a
+   * form of a cross) will be displayed.
```

Comment:
The lines appear in the source of NgbAlert (the "regular" alert component). This is a new input for NgbSelfClosingAlert, previously named NgbDismissibleAlert, which didn't have such an input: the close button was always visible in such alerts, despite the fact that they can close themselves automatically. 

So I added the equivalent input here and copy-pasted the api doc from NgbAlert, to be able to show or hide the close buttons on self-closing alerts too. I also chose to make the button invisible by default, since this component is self-closing and thus doesn't need a close button most of the time.

I'll amend the PR with "an ×" instead of "a cross".